src/main/resources/lang/lang_de_DE.properties

```diff
@@ -1017,6 +1017,18 @@ FactionDisbandedZeroPowerNotificationTitle=Fraktion aufgelï¿½st
 FactionDisbandedZeroPowerNotificationBody=Deine Fraktion, {0}, wurde aufgelï¿½st, da sie 0 Kraft erreicht hat.
 Teleporting=Teleportieren in {0} Sekunden...
 TeleportationCancelled=Teleportation abgebrochen.
-PowerIncreased=Du fï¿½hlst dich wieder stï¿½rker. Dein macht ist auf {0} gestiegen.
+PowerIncreased=Du f�hlst dich wieder st�rker. Dein macht ist auf {0} gestiegen.
```

Comment:
Umlauts are displaying strangely for me here

Reply:
Yeah those look strange. I might just install linux tonight and re-commit some changes on there (assuming the problem is an OS issue)